manifests/i/iamxiaojianzheng/jd-gui/1.6.8/iamxiaojianzheng.jd-gui.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: iamxiaojianzheng.jd-gui
+PackageVersion: 1.6.8
+Scope: machine
```

Comment:
`Scope` key has added to `installer.yaml`, but it still can't pass the pipeline. 😕